
# This is a patch for AcePerl-1.92 to update it to AcePerl-1.92-patched
# 
# To apply this patch:
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'applypatch' program with this patch file as input.
#
# If you do not have 'applypatch', it is part of the 'makepatch' package
# that you can fetch from the Comprehensive Perl Archive Network:
# http://www.perl.com/CPAN/authors/Johan_Vromans/makepatch-x.y.tar.gz
# In the above URL, 'x' should be 2 or higher.
#
# To apply this patch without the use of 'applypatch':
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'patch' program with this file as input.
#
#### End of Preamble ####

#### Patch data follows ####
diff -c "AcePerl-1.92\Makefile.PL" "AcePerl-1.92-patched\Makefile.PL"
Index: ./Makefile.PL
*** ./Makefile.PL	Wed Jan  4 08:18:10 2006
--- ./Makefile.PL	Wed Feb 10 20:39:45 2010
***************
*** 10,16 ****
  		  "  1) Interface to Ace socket server and local databases (pure Perl)\n" .
  		  "  2) The above plus XS optimizations (requires C compiler)\n" .
  		  "  3) The above plus RPC server interface (requires C compiler)\n\n" .
! 		  "Enter your choice: ", "1");
    if ($reply =~ /(\d+)/) {
      $choice = $1;
      die "invalid choice: $choice!" if $choice < 1  ||  $choice > 3;
--- 10,16 ----
  		  "  1) Interface to Ace socket server and local databases (pure Perl)\n" .
  		  "  2) The above plus XS optimizations (requires C compiler)\n" .
  		  "  3) The above plus RPC server interface (requires C compiler)\n\n" .
! 		  "Enter your choice: ", "2");
    if ($reply =~ /(\d+)/) {
      $choice = $1;
      die "invalid choice: $choice!" if $choice < 1  ||  $choice > 3;
#### End of Patch data ####

#### ApplyPatch data follows ####
# Data version        : 1.0
# Date generated      : Wed Feb 10 20:41:01 2010
# Generated by        : makepatch 2.04
# Recurse directories : Yes
# Excluded files      : (\A|/).*\~\Z
#                       (\A|/).*\.a\Z
#                       (\A|/).*\.bak\Z
#                       (\A|/).*\.BAK\Z
#                       (\A|/).*\.elc\Z
#                       (\A|/).*\.exe\Z
#                       (\A|/).*\.gz\Z
#                       (\A|/).*\.ln\Z
#                       (\A|/).*\.o\Z
#                       (\A|/).*\.obj\Z
#                       (\A|/).*\.olb\Z
#                       (\A|/).*\.old\Z
#                       (\A|/).*\.orig\Z
#                       (\A|/).*\.rej\Z
#                       (\A|/).*\.so\Z
#                       (\A|/).*\.Z\Z
#                       (\A|/)\.del\-.*\Z
#                       (\A|/)\.make\.state\Z
#                       (\A|/)\.nse_depinfo\Z
#                       (\A|/)core\Z
#                       (\A|/)tags\Z
#                       (\A|/)TAGS\Z
# p "Makefile.PL" 4740 1265859585 0100666
#### End of ApplyPatch data ####

#### End of Patch kit [created: Wed Feb 10 20:41:01 2010] ####
#### Patch checksum: 55 2253 9566 ####
#### Checksum: 73 2945 1336 ####
